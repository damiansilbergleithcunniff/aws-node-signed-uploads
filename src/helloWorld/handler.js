import helloWorld from './helloWorld';

const main = (event, context, callback) => {
  const response = helloWorld(200, 'You should rework this component', event);

  callback(null, response);
};

export default main;
