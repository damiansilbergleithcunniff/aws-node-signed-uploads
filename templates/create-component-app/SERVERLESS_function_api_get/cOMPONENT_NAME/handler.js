import cOMPONENT_NAME from './cOMPONENT_NAME';

const main = (event, context, callback) => {
  const response = cOMPONENT_NAME(200, 'You should rework this component', event);

  callback(null, response);
};

export default main;
