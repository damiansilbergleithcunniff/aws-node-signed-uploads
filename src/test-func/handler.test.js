import { hello } from './handler';

describe('Hello world function', () => {
  test('default snapshot matches', () => {
    let response;
    const event = "EVENT";
    const callback = (_, res) => { response = res };
    hello(event, null, callback);

    expect(response).toMatchSnapshot();
  });
});
