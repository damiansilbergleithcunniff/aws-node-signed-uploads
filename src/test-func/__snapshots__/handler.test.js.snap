// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello world function default snapshot matches 1`] = `
Object {
  "body": "{\\"message\\":\\"Go Serverless v1.0! Your function executed successfully!\\",\\"input\\":\\"EVENT\\"}",
  "statusCode": 200,
}
`;
